internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -276,9 +272,9 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 							MarkdownDescription: "Unique 24-hexadecimal digit string that identifies the zone in a Global Cluster. This value can be used to configure Global Cluster backup policies.",
 						},
 						"zone_name": schema.StringAttribute{
-							Computed:            true, // must be computed to have a Default
-							Optional:            true,
-							Default:             stringdefault.StaticString("ZoneName managed by Terraform"), // TODO: as in current resource
+							Computed: true, // must be computed to have a Default
```

Comment:
```suggestion
							Computed: true, 
```

i think it's clear for us now that we normally use Computed+Optional because of this

Reply:
Chagned